### PR TITLE
add customFields methods

### PR DESCRIPTION
**Pull request by bentona**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    ruby-pardot (1.3.0)
```
Comment (bentona) [inline Gemfile.lock:4]: Guessing this version mismatch is why I had to fix an extra test

File: spec/pardot/objects/custom_fields_spec.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+require File.expand_path(File.dirname(__FILE__) + '/../../spec_helper')
+
+describe Pardot::Objects::CustomFields do
+  
+  before do
+    @client = create_client
+  end
+  
+  describe "query" do
```
Comment (t-richards) [inline spec/pardot/objects/custom_fields_spec.rb:9]: `"#query"`

File: spec/pardot/objects/custom_fields_spec.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          </customField>
+        </result>
+      </rsp>)
+    end
+    
+    before do
+      @client = create_client
+    end
+    
+    it "should take in some arguments" do
```
Comment (t-richards) [inline spec/pardot/objects/custom_fields_spec.rb:35]: `it "takes in some arguments"`

Meh, the rest of the gem uses "should", so 🤷‍♂

File: spec/pardot/objects/custom_fields_spec.rb (hunk 2 above)
Comment (bentona) [reply spec/pardot/objects/custom_fields_spec.rb:9]: Yeah sadly that's the current idiom and I want to get this merged upstream so holding off on changes like that unless we decide tor really adopt it 100%

File: spec/pardot/objects/custom_fields_spec.rb (hunk 3 above)
Comment (bentona) [reply spec/pardot/objects/custom_fields_spec.rb:35]: :( yeah, same as above

**Review by tristandunn — APPROVED** [review]
Seems fine to me. Just don't let the cops see it.

**Pull request merged**